source/ios/AdaptiveCards/AdaptiveCards/AdaptiveCards/ACRLongPressGestureRecognizerFactory.mm

```diff
@@ -14,54 +14,67 @@
 #import "ToggleVisibilityAction.h"
 #import "ACRAggregateTarget.h"
 #import "ACOBaseActionElementPrivate.h"
+#import "ACRUILabel.h"
 
 using namespace AdaptiveCards;
 
 @implementation ACRLongPressGestureRecognizerFactory
 // instantiates a target for UILongPressGestureRecognizer object
 // and instantiate a tap gesture reconginizer with target, and return it
 // when failed, nil is returned
++ (NSObject<ACRSelectActionDelegate> *)buildTarget:(std::shared_ptr<BaseActionElement> const &)action
```

Comment:
>get: [](start = 47, length = 4)

nit: indenting looks off

Reply:
o.k.

---
In reply to: [272290199](https://github.com/Microsoft/AdaptiveCards/pull/2613#discussion_r272290199) [](ancestors = 272290199)